persistence/sql/persister_oauth2.go

```diff
@@ -361,30 +361,75 @@ func (p *Persister) RevokeAccessToken(ctx context.Context, id string) error {
 	return p.deleteSessionByRequestID(ctx, id, sqlTableAccess)
 }
 
-func (p *Persister) FlushInactiveAccessTokens(ctx context.Context, notAfter time.Time) error {
+func (p *Persister) FlushInactiveAccessTokens(ctx context.Context, notAfter time.Time, limit int, batchSize int) error {
```

Comment:
We want to remove this endpoint in the long run, so it is fine to not update this IMO or just have the defaults to make the API compatible. :)

Reply:
The janitor command calls this too, did you mean [this](https://github.com/ory/hydra/blob/b5b0791f495eda98235fbd8cff79f432dfb21653/oauth2/handler.go#L521-L543)?

Reply:
Oh sorry, I mistook this for the Flush HTTP endpoint. Scrap my comment!